### ISSUE TITLE
CSS Style Guide Issues

### ISSUE THREAD

**Issue by mcrickman** [wagtail/wagtail#7190]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

While reviewing the style guide when changing the  `:root { --color-primary-hue: 200.47619; }` (example)
The header button's style doesn't adjust to the change of the color due to being set to:

```css
header .button {
    background-color: #00676a;
    /* background-color: var(--color-primary-darker); */
}
```

the tabs and breadcrumbs are affected due to not using the css vars.

Also the color teal should be set to a teal color and not var(--color-primary)?

```css
.palette .color-teal {
    background-color: var(--color-primary);
}
```

### Steps to Reproduce

1. (for example) Start a new project with `wagtail start myproject`
2. enable the style guide and review style guide page
3. change the value in :root { --color-primary-hue: 180.4761904762;  } using chrome inspector and not the results.

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: (yes)

**Comment by thibaudcolas** [comment]
This will be fixed by #8824.

**Comment by thibaudcolas** [comment]
Fixed :)
